PowerEditor/installer/nsisInclude/uninstall.nsh

```diff
@@ -116,7 +133,6 @@ Section un.UserManual
 SectionEnd
 
 
-Var keepUserData
 Function un.doYouReallyWantToKeepData
 	StrCpy $keepUserData "false"
```

Comment:
Right, it it not required in our case.
But it is always to good to reset the variable (or handle both Yes and No case) in a function which is meant to set/reset based on user interaction. Such a kind of functions can be called multiple times, so it is MUST to reset the value otherwise the variable may carry previous value if user clicks "No". 

In our case also the function is called twice (but in the flow it will be called only once). Let me know if you want me to remove it.